Symfony/CS/Fixer/PSR2/SwitchCaseSpaceFixer.php

```diff
@@ -38,7 +38,7 @@ public function fix(\SplFileInfo $file, $content)
                     ++$ternariesCount;
                 }
 
-                if ($tokens[$colonIndex]->equals(':')) {
+                if ($tokens[$colonIndex]->equalsAny(array(':', ';'), false)) {
```

Comment:
the default is true, but it is not needed so false should be faster in execution doesn't it?